### PR TITLE
i18n(it): update translations for `reference/frontmatter`

### PR DESCRIPTION
**Pull request by itsmatteomanf**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

Updates translation of `reference/frontmatter.md`.

### CODE REVIEW

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 1]
```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
...
+    Abbiamo appena lanciato qualcosa di interessante!
+    <a href="https://example.com">Dai un'occhiata</a>
+---
+```
+
+### `lastUpdated`
+
+**type:** `Date | boolean`
+
+Sostituisce l'[opzione globale `lastUpdated`](/reference/configuration/#lastupdated). Se viene specificata una data, deve essere un [timestamp YAML](https://yaml.org/type/timestamp.html) valido e sovrascriverà la data archiviata nella cronologia Git per questa pagina.
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:160]: Qua possiamo usare un collegamento alla pagina `/it/`:

```suggestion
Sostituisce l'[opzione globale `lastUpdated`](/it/reference/configuration/#lastupdated). Se viene specificata una data, deve essere un [timestamp YAML](https://yaml.org/type/timestamp.html) valido e sovrascriverà la data archiviata nella cronologia Git per questa pagina.
```

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 2]
```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
...
+title: Pagina con una data di ultimo aggiornamento personalizzata
+lastUpdated: 2022-08-09
+---
+```
+
+### `prev`
+
+**type:** `boolean | string | { link?: string; label?: string }`
+
+Sostituisce l'[opzione globale `paginazione`](/reference/configuration/#pagination). Se viene specificata una stringa, il testo del collegamento generato verrà sostituito e se viene specificato un oggetto, sia il collegamento che il testo verranno sovrascritti.
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:173]: Lo stesso qua:

```suggestion
Sostituisce l'[opzione globale `paginazione`](/it/reference/configuration/#pagination). Se viene specificata una stringa, il testo del collegamento generato verrà sostituito e se viene specificato un oggetto, sia il collegamento che il testo verranno sovrascritti.
```

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 3]
```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
...
+# Nascondi il collegamento alla pagina successiva
+next: false
+---
+```
+
+### `sidebar`
+
+**type:** `{ label?: string; order?: number; hidden?: boolean; badge?: string | BadgeConfig }`
+
+Controlla il modo in cui questa pagina viene visualizzata nella [barra laterale](/reference/configuration/#sidebar), quando si utilizza un gruppo di collegamenti generato automaticamente.
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:215]: E pure qui 😁 
```suggestion
Controlla il modo in cui questa pagina viene visualizzata nella [barra laterale](/it/reference/configuration/#sidebar), quando si utilizza un gruppo di collegamenti generato automaticamente.
```

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 4]
```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
...
+---
+```
+
+#### `badge`
+
+**type:** <code>string | <a href="/reference/configuration/#badgeconfig">BadgeConfig</a></code>
+
+Aggiungi un badge alla pagina nella barra laterale quando viene visualizzata in un gruppo di collegamenti generato automaticamente.
+Quando si utilizza una stringa, il badge verrà visualizzato con un colore in risalto predefinito.
+Facoltativamente, passa un [oggetto `BadgeConfig`](/reference/configuration/#badgeconfig) con i campi `text` e `variant` per personalizzare il badge.
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:268]: E per ultimo qua:

```suggestion
Facoltativamente, passa un [oggetto `BadgeConfig`](/it/reference/configuration/#badgeconfig) con i campi `text` e `variant` per personalizzare il badge.
```

File: docs/src/content/docs/it/reference/frontmatter.md [hunk 5]
```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
...
+---
+title: Pagina con un badge
+sidebar:
+  # Utilizza la variante predefinita corrispondente al colore principale del tuo sito
+  badge: nuovo
+---
+```
+
+```md
+
```
Comment (delucis) [inline docs/src/content/docs/it/reference/frontmatter.md:280]: Sembra che mancano i contenuti qua:

```suggestion
---
title: Pagina con un badge
sidebar:
  badge:
    text: Esperimentale
    variant: caution
---
```

**Review by delucis — COMMENTED** [review]
Grazie per i aggiornamenti @itsmatteomanf! Ho un paio di commenti, però dopo questi, sembra ottimo.

File: docs/src/content/docs/it/reference/frontmatter.md (hunk 1 above)
Comment (itsmatteomanf) [reply docs/src/content/docs/it/reference/frontmatter.md:160]: You are right! I did it in the normal Astro docs which had the `/en/`, totally forgot it here as it was root (why the difference, though?). It's gonna be same in most PRs, will update them all later!

**Comment by itsmatteomanf** [comment]
Grazie a te, @delucis!

Will update the other PRs which are gonna have the same issue.

File: docs/src/content/docs/it/reference/frontmatter.md (hunk 1 above)
Comment (delucis) [reply docs/src/content/docs/it/reference/frontmatter.md:160]: The ability to use a root locale is something we got a lot in relation to i18n so it seemed like we needed to support it for a general tool like Starlight even though it does add some code complexity. We use it for the Starlight docs to dog food it and catch bugs.

Comment (itsmatteomanf) [reply docs/src/content/docs/it/reference/frontmatter.md:160]: Oh, gotcha! Makes sense then.

It's annoying for the various imports as they change the depth when you move them over to any language ahah I'm tempted to make a PR to use aliases.

**Comment by itsmatteomanf** [comment]
Ok, tutte sistemate (credo e spero ahah) anche nelle altre PR.

QUando vuoi, procedi pure @delucis :)

**Review by delucis — APPROVED** [review]
Ottimo! Grazie per la rielaborazione 🙌

**Pull request merged**